nx-libs.spec

```diff
@@ -804,23 +771,6 @@ ln -s -f ../../../../%{_lib}/libXext.so.6 %{buildroot}%{_libdir}/nx/X11/Xinerama
 %defattr(-,root,root)
 %{_libdir}/libNX_Xau.so.6*
 
-%files -n libNX_Xcomposite1
-%defattr(-,root,root)
-%{_libdir}/libNX_Xcomposite.so.1*
-
-%files -n libNX_Xdamage1
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdamage.so.1*
-
-%files -n libNX_Xdmcp-devel
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdmcp.so
-%{_includedir}/nx/X11/Xdmcp.h
-
-%files -n libNX_Xdmcp6
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdmcp.so.6*
-
```

Comment:
In Debian/Ubuntu, the dummy packages are a clear workflow element for autoremoving packages on dist-upgrades.

I have no clue, how RPM based distros handle this, but it seems logic to provide an upgrade path to an empty package that has no other (forward + reverse) dependencies. So that the actual obsolete files (shipped with previous package versions) get removed from the system.

Another option might be the "Obsoletes:" field in .spec.files. But as said before, I have no clue.